ftml/src/preproc/typography.rs

```diff
@@ -189,14 +196,41 @@ const TEST_CASES: [(&str, &str); 3] = [
         "**ENTITY MAKES DRAMATIC MOTION** . . . ",
         "**ENTITY MAKES DRAMATIC MOTION** … ",
     ),
+    ("Whales... they are cool", "Whales… they are cool"),
+    ("Whales ... they are cool", "Whales … they are cool"),
+    ("...why would you think that?", "…why would you think that?"),
+    ("how could you...", "how could you…"),
+    (
+        "... why would you think that?",
+        "… why would you think that?",
+    ),
+    ("how could you ...", "how could you …"),
+    ("Whales. . . they are cool", "Whales… they are cool"),
+    (
+        ". . .why would you think that?",
+        "…why would you think that?",
+    ),
+    ("how could you. . .", "how could you…"),
+    ("Whales . . . they are cool", "Whales … they are cool"),
+    (
+        ". . . why would you think that?",
+        "… why would you think that?",
+    ),
+    ("how could you . . .", "how could you …"),
+    (".... ..", ".... .."),
+    (". . .. ....", ". . .. ...."),
+    ("... . . . . . .", "… … …"),
+    ("..........", ".........."),
+    ("... ... ...", "… … …"),
+    ("... . . . ...", "… … …"),
```

Comment:
Then for each of these dot tests, you could include a brief prefix (or code comment) to denote its purpose. FOr instance:
```rust
"four dots .... ..",
"only two . . .. ....",
// or
"..........", // continuous dots
"... ... ...", // groups of three
```

Reply:
I chose to leave comments, are they descriptive enough?